.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
Will it work for subfolders like `modules/8_reinforcement_learning/__pycache__`?

Reply:
Yes, it will.